lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -55,7 +55,9 @@ function collectFormElements() {
         id: child.id,
         name: child.name,
         placeholder: child instanceof HTMLSelectElement ? undefined : child.placeholder,
-        autocomplete: child.autocomplete,
+        autocompleteProp: child.autocomplete,
```

Comment:
can we do `autocomplete: {property: string, attribute: string, prediction: string}` ?

Reply:
makes sense, on it